bot/cogs/commands/minecraft.py

```diff
@@ -155,50 +156,39 @@ async def mcstatus(self, ctx: Ctx, host=None, port: int = None):
 
             combined = f"{host}{port_str}"
 
-        fail = False
-        jj: dict = None
-
-        async with SuppressCtxManager(ctx.typing()):
```

Comment:
Why was this `SuppressCtxManager(ctx.typing())` removed? I think it's nice to show that the bot is "thinking" (while fetching the status)

Reply:
https://github.com/Iapetus-11/Villager-Bot/pull/277/commits/97f04e3a9e55ebfc5e3b8f38955b9f18102099a8